### ISSUE TITLE
ide create fails whilst encrypting master password

### ISSUE THREAD

**Issue by hohwille** [devonfw/IDEasy#573]
I found another bocker preventing to create a new project with the latest IDEasy release:

```
$ ide create newprj -
Creating new IDEasy project in D:\projects\newprj
Start: Clone settings repository
Cloning into '.'...
remote: Enumerating objects: 1143, done.
remote: Counting objects: 100% (571/571), done.
remote: Compressing objects: 100% (329/329), done.
remote: Total 1143 (delta 249), reused 488 (delta 197), pack-reused 572 (from 1)
Receiving objects: 100% (1143/1143), 219.36 KiB | 1.73 MiB/s, done.
Resolving deltas: 100% (384/384), done.
Successfully updated settings repository.
Start: Copy configuration templates
Copying template D:\projects\newprj\settings\templates\conf\ide.properties to D:\projects\newprj\conf.
Copying template D:\projects\newprj\settings\templates\conf\npm\.npmrc to D:\projects\newprj\conf\npm.
Copy configuration templates
Start: Install or update software
Successfully installed java in version 22.0.2_9
Start: Install mvn
Start: Create mvn settings security file at D:\projects\newprj\conf\mvn\settings-security.xml
Step 'Create mvn settings security file at D:\projects\newprj\conf\mvn\settings-security.xml' ended with failure.
Step 'Install or update software' ended with failure.
Step 'ide (create,newprj,-)' failed: java.lang.IllegalStateException: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed with exit code 1!
 Step 'Install or update software' failed: null
  Step 'Install mvn (*)' failed: java.lang.IllegalStateException: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed with exit code 1!
   Step 'Create mvn settings security file at D:\projects\newprj\conf\mvn\settings-security.xml' failed: null
4 step(s) failed out of 7 steps.
An unexpected error occurred!
We are sorry for the inconvenience.
Please check the error below, resolve it and try again.
If the error is not on your end (network connectivity, lack of permissions, etc.) please file a bug:
https://github.com/devonfw/ide/issues/new?assignees=&labels=bug&projects=&template=bug.md&title=IllegalStateException%3A+Running+command+%27D%3A%5Cprojects%5C_ide%5Csoftware%5Cdefault%5Cmvn%5Cmvn%5C3.9.9%5Cbin%5Cmvn.cmd%27+with+arguments+%27--encrypt-master-password%27+%271B%2BU76nWsK2MQMejrSls46xrn5Q%3D%27+failed%3A+Running+command+%27D%3A%5Cprojects%5C_ide%5Csoftware%5Cdefault%5Cmvn%5Cmvn%5C3.9.9%5Cbin%5Cmvn.cmd%27+with+arguments+%27--encrypt-master-password%27+%271B%2BU76nWsK2MQMejrSls46xrn5Q%3D%27+failed+with+exit+code+1%21
java.lang.IllegalStateException: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed with exit code 1!
        at com.devonfw.tools.ide.process.ProcessContextImpl.run(ProcessContextImpl.java:173)
        at com.devonfw.tools.ide.tool.mvn.Mvn.createSettingsSecurityFile(Mvn.java:109)
        at com.devonfw.tools.ide.tool.mvn.Mvn.postInstall(Mvn.java:88)
        at com.devonfw.tools.ide.tool.LocalToolCommandlet.doInstall(LocalToolCommandlet.java:97)
        at com.devonfw.tools.ide.tool.plugin.PluginBasedCommandlet.doInstall(PluginBasedCommandlet.java:145)
        at com.devonfw.tools.ide.tool.ToolCommandlet.install(ToolCommandlet.java:180)
        at com.devonfw.tools.ide.tool.mvn.Mvn.install(Mvn.java:72)
        at com.devonfw.tools.ide.commandlet.AbstractUpdateCommandlet.updateSoftware(AbstractUpdateCommandlet.java:170)
        at com.devonfw.tools.ide.commandlet.AbstractUpdateCommandlet.run(AbstractUpdateCommandlet.java:52)
        at com.devonfw.tools.ide.commandlet.CreateCommandlet.run(CreateCommandlet.java:61)
        at com.devonfw.tools.ide.context.AbstractIdeContext.applyAndRun(AbstractIdeContext.java:902)
        at com.devonfw.tools.ide.context.AbstractIdeContext.run(AbstractIdeContext.java:830)
        at com.devonfw.tools.ide.cli.Ideasy.runOrThrow(Ideasy.java:88)
        at com.devonfw.tools.ide.cli.Ideasy.run(Ideasy.java:51)
        at com.devonfw.tools.ide.cli.Ideasy.main(Ideasy.java:28)
        at java.base@21.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Caused by: com.devonfw.tools.ide.cli.CliException: Running command 'D:\projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' '1B+U76nWsK2MQMejrSls46xrn5Q=' failed with exit code 1!
        at com.devonfw.tools.ide.process.ProcessContextImpl.performLogOnError(ProcessContextImpl.java:289)
        at com.devonfw.tools.ide.process.ProcessContextImpl.run(ProcessContextImpl.java:164)
        ... 15 more


Error: IDEasy failed with exit code 255
```

**Comment by hohwille** [comment]
For the record:
```
$ ide mvn -v
Apache Maven 4.0.0-beta-3 (e92f645c2749eb2a4f5a8843cf01e7441e4b559f)
Maven home: D:\projects\IDEasy\software\mvn
Java version: 21.0.4, vendor: Eclipse Adoptium, runtime: D:\projects\IDEasy\software\java
Default locale: en_US, platform encoding: UTF-8
OS name: "windows 11", version: "10.0", arch: "amd64", family: "winnt"
Successfully completed ide (mvn,-v)
```

so it has nothing to do with the fact that IDEasy is calling `mvn.bat`.

**Comment by ndemirca** [comment]
I couldn't reproduce this bug. Could you recheck it please @hohwille ?

**Comment by hohwille** [comment]
Sure, happens all the time, when I do `ide create «newprojectname»`.

**Comment by hohwille** [comment]
Seems to be a duplicate of #592

**Comment by ndemirca** [comment]
Hi, here are my debugs. I hope it is helpful. 

 ide -d create test
Configuration directory C:\Users\ndemirca\.ide does not exist.
System console not available - using System.in as fallback
Running commandlet CreateCommandlet[create]
Creating new IDEasy project in C:\Projects\test
Configuration directory C:\Users\ndemirca\.ide does not exist.
Configuration directory C:\Projects\test\settings does not exist.
Configuration directory C:\Projects\test\conf does not exist.
Start: Clone settings repository
Missing your settings at C:\Projects\test\settings and no SETTINGS_URL is defined.
Further details can be found here: https://github.com/devonfw/IDEasy/blob/main/documentation/settings.asciidoc
Please contact the technical lead of your project to get the SETTINGS_URL for your project.
In case you just want to test IDEasy you may simply hit return to install the default settings.
Settings URL [https://github.com/devonfw/ide-settings.git]:

Running command 'C:\Program Files\Git\mingw64\bin\git.exe' with arguments 'clone' '--recursive' 'https://github.com/devonfw/ide-settings.git' '--config' 'core.autocrlf=false' '.' ...
Cloning into '.'...
remote: Enumerating objects: 1153, done.
remote: Counting objects: 100% (579/579), done.
remote: Compressing objects: 100% (336/336), done.
remote: Total 1153 (delta 252), reused 493 (delta 198), pack-reused 574 (from 1)
Receiving objects: 100% (1153/1153), 222.19 KiB | 15.87 MiB/s, done.
Resolving deltas: 100% (388/388), done.
Successfully updated settings repository.
Step 'Clone settings repository' ended successfully.
Start: Copy configuration templates
Copying template C:\Projects\test\settings\templates\conf\ide.properties to C:\Projects\test\conf.
Copying C:\Projects\test\settings\templates\conf\ide.properties recursively to C:\Projects\test\conf\ide.properties
Copying template C:\Projects\test\settings\templates\conf\npm\.npmrc to C:\Projects\test\conf\npm.
Copying C:\Projects\test\settings\templates\conf\npm\.npmrc recursively to C:\Projects\test\conf\npm\.npmrc
Copy configuration templates
Step 'Copy configuration templates' ended successfully.
Start: Install or update software
Tool node not installed in C:\Projects\test\software\node
In git repository C:\Projects\_ide\urls the timestamp file HEAD was last updated PT2M26.123S ago and caching duration in PT30M.
Invalid log message with 3 argument(s) but less placeholders: Skipping git {} on {} because last fetch was just recently to avoid overhead.
Skipping git pull/clone on C:\Projects\_ide\urls because last fetch was just recently to avoid overhead.
Running command 'C:\Program Files\Git\mingw64\bin\git.exe' with arguments 'diff-index' '--quiet' 'HEAD' ...
Running command 'C:\Program Files\Git\mingw64\bin\git.exe' with arguments 'ls-files' '--other' '--directory' '--exclude-standard' ...
Resolved version pattern * to version v22.8.0
Artifact already exists at C:\Users\ndemirca\Downloads\ide\default\node-v22.8.0-windows-x64.zip
To force update please delete the file and run again.
Checksum d6e1c4fca93997224cac0bec09b4201aa018f50171d38c6b85abe483012839c9 is correct.
Unpacking 100% [=======================] 31/31MiB (0:00:03 / 0:00:00) 10.6MiB/s
Deleting C:\Projects\_ide\tmp\extract-node-v22.8.0-windows-x64.zip ...
Successfully installed node in version v22.8.0
Step 'Install node' ended successfully.
Tool npm not installed in C:\Projects\test\software\npm
Start: Install npm
Resolved version pattern * to version 10.8.3
Version 10.8.3 of tool npm is already installed at C:\Projects\_ide\software\default\npm\npm\10.8.3
Due to lack of permissions, Microsoft's mklink with junction had to be used to create a Symlink. See https://github.com/devonfw/IDEasy/blob/main/documentation/symlinks.asciidoc for further details. Error was: C:\Projects\test\software\npm: A required privilege is not held by the client
You are on Windows and you do not have permissions to create symbolic links. Junctions are used as an alternative, however, these can not point to relative paths. So the source (..\..\_ide\software\default\npm\npm\10.8.3) is interpreted as an absolute path.
Running command 'C:\WINDOWS\system32\cmd.exe' with arguments '/c' 'mklink' '/d' '/j' 'C:\Projects\test\software\npm' 'C:\Projects\_ide\software\default\npm\npm\10.8.3' ...
Junction created for C:\Projects\test\software\npm <<===>> C:\Projects\_ide\software\default\npm\npm\10.8.3
Successfully installed npm in version 10.8.3
Step 'Install npm' ended successfully.
Tool java not installed in C:\Projects\test\software\java
Resolved version pattern * to version 22.0.2_9
Version 22.0.2_9 of tool java is already installed at C:\Projects\_ide\software\default\java\java\22.0.2_9
Due to lack of permissions, Microsoft's mklink with junction had to be used to create a Symlink. See https://github.com/devonfw/IDEasy/blob/main/documentation/symlinks.asciidoc for further details. Error was: C:\Projects\test\software\java: A required privilege is not held by the client
You are on Windows and you do not have permissions to create symbolic links. Junctions are used as an alternative, however, these can not point to relative paths. So the source (..\..\_ide\software\default\java\java\22.0.2_9) is interpreted as an absolute path.
Running command 'C:\WINDOWS\system32\cmd.exe' with arguments '/c' 'mklink' '/d' '/j' 'C:\Projects\test\software\java' 'C:\Projects\_ide\software\default\java\java\22.0.2_9' ...
Junction created for C:\Projects\test\software\java <<===>> C:\Projects\_ide\software\default\java\java\22.0.2_9
Successfully installed java in version 22.0.2_9
Step 'Install java' ended successfully.
Tool mvn not installed in C:\Projects\test\software\mvn
Start: Install mvn
Resolved version pattern * to version 3.9.9
Version 3.9.9 of tool mvn is already installed at C:\Projects\_ide\software\default\mvn\mvn\3.9.9
Due to lack of permissions, Microsoft's mklink with junction had to be used to create a Symlink. See https://github.com/devonfw/IDEasy/blob/main/documentation/symlinks.asciidoc for further details. Error was: C:\Projects\test\software\mvn: A required privilege is not held by the client
You are on Windows and you do not have permissions to create symbolic links. Junctions are used as an alternative, however, these can not point to relative paths. So the source (..\..\_ide\software\default\mvn\mvn\3.9.9) is interpreted as an absolute path.
Running command 'C:\WINDOWS\system32\cmd.exe' with arguments '/c' 'mklink' '/d' '/j' 'C:\Projects\test\software\mvn' 'C:\Projects\_ide\software\default\mvn\mvn\3.9.9' ...
Junction created for C:\Projects\test\software\mvn <<===>> C:\Projects\_ide\software\default\mvn\mvn\3.9.9
Start: Create mvn settings security file at C:\Projects\test\conf\mvn\settings-security.xml
Running command 'C:\Projects\_ide\software\default\mvn\mvn\3.9.9\bin\mvn.cmd' with arguments '--encrypt-master-password' 'ogogzvHvxYBEwge3BArk8/KXQT0=' ...
Create mvn settings security file at C:\Projects\test\conf\mvn\settings-security.xml
Step 'Create mvn settings security file at C:\Projects\test\conf\mvn\settings-security.xml' ended successfully.
Start: Create mvn settings file at C:\Projects\test\conf\mvn\settings.xml
Running command 'C:\Program Files\Git\mingw64\bin\git.exe' with arguments '-C' 'C:\Projects\test\settings' 'remote' '-v' ...
Create mvn settings file at C:\Projects\test\conf\mvn\settings.xml
Step 'Create mvn settings file at C:\Projects\test\conf\mvn\settings.xml' ended successfully.
Successfully installed mvn in version 3.9.9
Step 'Install mvn' ended successfully.
Install or update software
Step 'Install or update software' ended successfully.
Importing repository from IDEasy.properties ...
Repository is not active by default.
Skipping repository - use force (-f) to setup all repositories ...
Successfully created new project 'test'.
Step 'ide' ended successfully.
Successfully completed ide (create,test)
You are not inside an IDE installation: C:\Projects
